### ISSUE TITLE
Some BGM/sound effect versions are inconsistent with the question arcs

### ISSUE THREAD

**Issue by 0xC0FFEE-Mug** [07th-mod/meakashi#68]
Hello 07th-mod, first of all thank you very much for creating this project!

I completed episode 1-4 and started with episode 5 today, I couldn't help but notice that some BGM/SE seem to be using what I'm guessing is the original version instead of the newer one I'm by now used to hearing.

To verify I didn't screw up the settings during installation, here some excerpts from the log (I could send the full log file over email or something as it I don't want to post it here as it might contain some info on my PC/paths):
```
> Install Started On 2020-10-24 17:44:43.315947
Python 3.7.7 (tags/v3.7.7:d7c567b08f, Mar 10 2020, 09:44:33) [MSC v.1900 32 bit (Intel)]
Installer Build Information: Build Date: 2020-09-08 02:39:09.302395
Git Tag (Version): refs/tags/v1.1.56

...

User selected options for install:
Option ID: [BGM Options: No BGM Changes] Value: [False]
Option ID: [BGM Options: BGM Fix] Value: [True]
Option ID: [SE Options: No SE Changes] Value: [False]
Option ID: [SE Options: SE Fix] Value: [True]
Option ID: [Alternate Languages: Apply French Language Patch] Value: [False]

...

Local Version: None
Remote Version: id: Meakashi Ch.5/full lastAttemptedInstallID: None files:
 - graphics-2.0.0
 - movie-1.0.0
 - voices-1.1.0
 - script-5.0.0
 - movie-unix-1.0.0
 - ui-windows-5.5.3p3-1.4.0
 - ui-windows-5.5.3p1-1.4.0
 - ui-unix-5.5.3p3-1.4.0
 - ui-unix-5.5.3p1-1.4.0
Full Update: True (5/5) excluding mod options
```

I did some comparisons in the BGM/SE dirs + game scripts and found the following:
* `wa_022.ogg` sounds like the updated version of `s_z01_iwakan.ogg`, but `s_z01` is still used at [this line](https://github.com/07th-mod/meakashi/blob/052807625c8cacc91526fd9aa66b5886ab5d882b/Update/_meak_002.txt#L1681). I haven't grepped to see if this is a one-time thing or everywhere throughout the episode/scripts (as I wouldn't want to get spoiled :-p).
* I conject this may also apply to `wa_023.ogg`/`s_z02_kyousou.ogg` and `wa_024.ogg`/`s_z03_haikyo.ogg`
* `wa_38.ogg` (the menu button clicky sound) differs
* Some BGMs sound different, as was pointed out in issue #49 (I've only noticed/recognised `msys27.ogg` so far)

Please note that this is by no means complete as I'm only around the start of chapter 2.

**Comment by enumag** [comment]
Thanks for reporting. We're not directly looking into the exact issues you listed but it triggered us to revive an older work in progress initiative about fixing BGMs in answer arcs. Your issues might be fixed by that as well although I didn't check. We're working on it in https://github.com/07th-mod/meakashi/pull/69 (requires some additional files and we're still internally discussing a few things).

**Comment by 0xC0FFEE-Mug** [comment]
Neat, thanks a lot for looking into it (and so quickly as well)!

>Your issues might be fixed by that as well although I didn't check.

I checked PR #69's `_meak_002.txt` (the chapter I initially noticed this in) and don't see changes to `s_z01_iwakan.ogg`/`wa_022.ogg` references in there, so I speculate that it's still present after that PR is merged.

I could very well be off though as I'm looking at this without knowing the inner workings of the game engine. I'm assuming the string argument to `PlaySE` relates directly to the SE file itself :-p

If I'm able to provide you with any more info I'll be glad to help.

**Comment by 0xC0FFEE-Mug** [comment]
Currently playing through Tsumihoroboshi and I think the same applies there as well. I'd imagine you wouldn't want the ticket duped there though :-p

**Comment by 0xC0FFEE-Mug** [comment]
The same seems to apply for Minagoroshi as well.

**Comment by drojf** [comment]
lrysa from Discord has prepared this table of files which potentially had issues, which I've summarized here. This seems to match up with what 0xC0FFEE-Mug said earlier.

Name of File | Location of File | Status of File | Suggested Replacement File
-- | -- | -- | --
wa_022.ogg | Question OLD SE fix | Incorrect | s_z01_iwakan.ogg from Answer OLD SE fix
wa_023.ogg | Question OLD SE fix | Incorrect | !s_z02_kyousou.ogg from Answer OLD SE fix
wa_024.ogg | Question OLD SE fix | Incorrect | s_z03_haikyo.ogg from Answer OLD SE fix

lrysa also had the following note: **"Files in Answer OLD SE fix seem to be using incorrect names compared to what was used in the original games, but the calls for the SEs are correct."**

To be clear, these are referencing the files in our [BGM fix and SE fix archives](https://07th-mod.com/wiki/Higurashi/Manual-Installation/#bgm-and-sound-effect-patches), not the files from the base game.

----

I should also mention for others reading this thread, that we have this (probably outdated) list of music changes. Please let me know if you'd like write access to this document: https://docs.google.com/spreadsheets/d/1c4Ou9ssNuTAsxJozD0nnc-fzfaIwAdk30F2JhhQGmw4/edit#gid=844796254

----

Finally, a reminder to myself that I should add a disclaimer about se/bgm fix having problems to the wiki/installer

- [ ] Add a disclaimer that the se/bgm fix still has some problems to the wiki
- [ ] Add a disclaimer that the se/bgm fix still has some problems to the installer

**Comment by drojf** [comment]
GIN and Irysa have gone through and created a fix for the SE (not yet applied). 

> Okay, I've compiled fixed SE files for answer arcs that should solve the Meakashi issue mentioned above. These five files should just overwrite those in the Answer Arcs SE pack. It should apply to all answer arcs.

[Fixed SE.zip](https://github.com/07th-mod/meakashi/files/5798885/Fixed.SE.zip)

The following files will be updated:

Name of File |
-- |
s_chaim.ogg  |
s_z01_iwakan.ogg |
s_z02_kyousou.ogg |
s_z03_haikyo.ogg |
wa_040.ogg |

- [ ] Update the Answer arcs SE pack

**Comment by drojf** [comment]
OK, I've gone and updated the answer arc SE pack with the above changes

The new se pack is here:
https://07th-mod.com/rikachama/audio/Higurashi-OldSE.7z

The previous version is here for comparison
https://07th-mod.com/rikachama/audio/HigurashiKai-OldSE-backup-2021-01-23.7z

Let me know if this fixes the problem/makes the packs consistent with each other, then we can close the issue.

**Comment by 0xC0FFEE-Mug** [comment]
Currently still playing through ch. 8 (for the first time).
Once I've completed that I'll give meakashi a reinstall / check it.

**Comment by drojf** [comment]
The Question Arcs BGM/SE we use in the Answer Arcs when you have 'New BGM' selected should now be identical to the Question arcs (they should be consistent).

I've sorted out the Answer Arcs BGM, so they all come from the Answer Arcs BGM archive, in the `ExtraBGM` and `ExtraSE` folder: https://07th-mod.com/rikachama/audio/HigurashiKai-OG-BGM-SE.7z . This same archive is used for chapters 5-8.

I cross-checked these BGM/SE with a file comparer and they are now identical to the unmodded game's BGM (I copied them from there myself, but just to double check I compared them)

I also removed any BGM/SE from any other archives where they shouldn't be, see https://github.com/07th-mod/meakashi/issues/49#issuecomment-1304706305. But even if there are some misplaced BGM, the ExtraBGM/ExtraSE folder should override them.
